libshaderc/include/shaderc/shaderc.h

```diff
@@ -74,6 +74,13 @@ typedef enum {
   shaderc_compilation_status_invalid_assembly,
 } shaderc_compilation_status;
 
+// Optimization level.
+typedef enum {
+  shaderc_optimization_level_none = 0,  // no optimization
+  shaderc_optimization_level_zero,      // no optimization
```

Comment:
It's just a conceptual difference; none means not specify any optimization level at all. But that may be too nuanced. I'll remove it.